sphinxemoji/codes.json

```diff
@@ -1243,7 +1243,7 @@
     ":gabon:": "🇬🇦",
     ":gambia:": "🇬🇲",
     ":game_die:": "🎲",
-    ":gay_pride_flag:": "🏳🌈",
+    ":gay_pride_flag:": "🏳️‍🌈",
```

Comment:
I am just slightly worried about this change, since this list is autogenerated. So the fix will be overwritten in the next update. Would you mind checking the list in [gemojione](https://github.com/bonusly/gemojione) and maybe suggest a fix there?

Reply:
I believe this has been addressed by the new version of the library. I rebased the branch, so the change should have disappeared. I updated the PR title not to mention the flag.